homeassistant/components/mopeka/manifest.json

```diff
@@ -63,5 +63,5 @@
   "documentation": "https://www.home-assistant.io/integrations/mopeka",
   "integration_type": "device",
   "iot_class": "local_push",
-  "requirements": ["mopeka-iot-ble==0.7.0"]
+  "requirements": ["mopeka-iot-ble==0.8.0"]
```

Comment:
Please bump dep in a separate pr

Reply:
so I should not change the integration manifest dependency in this PR?  The integration changes require the new package version to build and once I change the manifest will not check in without the manifest due to the build script blocking the commit.   

Sorry, I'm new to this show so I'm not clear if you don't want me to update the manifest or your just asking for a separate PR to identify the requirement to bump.

Reply:
never mind  i think i understand, will do.

Reply:
example https://github.com/home-assistant/core/pull/122704